core/node/node_framework/src/implementations/layers/da_dispatcher.rs

```diff
@@ -37,7 +37,7 @@ impl WiringLayer for DataAvailabilityDispatcherLayer {
 
     async fn wire(self: Box<Self>, mut context: ServiceContext<'_>) -> Result<(), WiringError> {
         let master_pool_resource = context.get_resource::<PoolResource<MasterPool>>().await?;
-        let master_pool = master_pool_resource.get().await?;
+        let master_pool = master_pool_resource.get_custom(2).await?;
```

Comment:
Nit: Please add a comment explaining why 2 connections are necessary. Optionally, copy to `zksync_core_leftovers` (since that crate is hopefully going to be removed soon, this is not strictly necessary).